scripts/python_utils_test.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for feature detection utilities for Python 2 and Python 3."""
+
+import StringIO
+import ast
+import io
+import sys
+
+from core.tests import test_utils
+
+from . import python_utils
+
+
+class PythonUtilsTests(test_utils.GenericTestBase):
```

Comment:
Perhaps there should be some tests somewhere for how string/unicode are handled. E.g. reading files with unicode chars (you can put an example in core/tests/data).

Reply:
I am sorry but there seems to be json and audio files in core/tests/data. What am I supposed to put there?

Reply:
You are welcome to put arbitrary files in core/tests/data that you want to use in tests. I was just mentioning it as a possible location if you wanted someplace to store the test data.

Also, it's fine not to use core/tests/data at all, if you can think of another way to test string/unicode chars in files.

Reply:
Done